### ISSUE TITLE
Thank you for your hard work and I'm sorry to hear the bad news

### ISSUE THREAD

**Issue by MartinX3** [k0shk0sh/FastHub#2826]
@k0shk0sh 
I wish you and your mom the best and a hopefully happy future.
I'm sorry to hear the bad news and want to thank you for your hard work!

**Comment by ron159** [comment]
Take care of yourself ❤️

**Comment by nitrog0d** [comment]
What bad news?

**Comment by nitrog0d** [comment]
Just updated the app, sorry for being slow.

**Comment by aniforprez** [comment]
Yeah goddamn I'm really sorry to hear the bad news. Hope things look up and are better for you

**Comment by droidmonkey** [comment]
Your efforts make it possible for me to control my project anywhere I am. Very much appreciate everything you do. We are here for you if needed.
